README.md

```diff
@@ -216,7 +218,37 @@ For a more in-depth understanding of the structure of these records, please expl
 In addition to the primary feature of interacting with Chainlink services, this plugin provides the ability to manage a local Chainlink node.
 The corresponding functionality is implemented in a special tasks module `sandbox`.
 This module implements methods for starting, restarting and stopping a Chainlink node, getting Chainlink node information,
-deploying and interacting with such contracts as [LinkToken](contracts%2FLinkToken.sol), [Operator](contracts%2FOperator.sol) and [ChainlinkDirectRequestConsumer](contracts%2FChainlinkDirectRequestConsumer.sol).
+deploying and interacting with such contracts as [LinkToken](contracts%2FLinkToken.sol), [Operator](contracts%2FOperator.sol), [FunctionsConsumer.sol](contracts%2FFunctionsConsumer.sol) and [ChainlinkDirectRequestConsumer](contracts%2FChainlinkDirectRequestConsumer.sol).
+
+### Configure and run Functions requests simulations
+
+This plugin allows you to run local Functions requests simulations.
```

Comment:
Suggested change:
`This plugin allows you to run local Functions requests simulations. A simulation is an execution of your custom JavaScript code in a locally spun up Deno sandbox environment. It is useful for debugging and for checking whether the source code you supply to Chainlink Functions can reasonably be expected to work when passed on-chain'